### PR TITLE
New file extension

### PR DESCRIPTION
**Pull request by cartifon**
Adding another valid extension to mustache files to highlight the tags.

### CODE REVIEW

File: grammars/mustache.cson [hunk 1]
```diff
@@ -5,6 +5,7 @@
   'hbs'
   'hjs'
   'mu'
+  'mst'
```
Comment (50Wliu) [inline grammars/mustache.cson]: Alphabetize this please.

**Comment by cartifon** [comment]
Done

**Comment by 50Wliu** [comment]
Thanks!

**Comment by cartifon** [comment]
No problem, thank you.

**Pull request merged**